central/notifications/datastore/internal/writer/writer_test.go

```diff
@@ -0,0 +1,193 @@
+package writer
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	storeMocks "github.com/stackrox/rox/central/notifications/datastore/internal/store/mocks"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/mock/gomock"
+)
+
+var errFake = errors.New("fake error")
+
+func TestNotificationsWriter(t *testing.T) {
```

Comment:
I'd say for the writer specifically, it'd be great to use integration tests with postgres instead of relying on the mocks.

Here's [also an example on how to initialize everything](https://github.com/stackrox/stackrox/blob/master/central/declarativeconfig/health/datastore/datastore_impl_test.go#L41-L44).